srcpkgs/gnome-usage/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'gnome-usage'
+pkgname=gnome-usage
+version=3.28.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="gettext-devel glib-devel gtk+3-devel libgtop-devel vala-devel"
```

Comment:
see if libglib-devel can be used instead. If a binary from `glib-devel` needs to be run then add `glib-devel` to hostmakedepends

Reply:
Oh, yeah, it needs glib-compile-schemas, that's why I didn't use `libglib-devel`

Reply:
Uhm...I guess this still need libglib-devel as makedepend though?